### DISCUSSION TITLE
Designing a new, fresh, and easy to use UI for dupeguru

### DISCUSSION THREAD

**Discussion by VasigaranAndAngel** [hsoft/dupeguru#1166] [Ideas]
Hello,

I believe that the current UI could be improved to make it more modern, fresh, and easy to use.

I want to start a discussion on how we can improve the UI of dupeGuru. What are your thoughts on this? Would you happen to have any suggestions or ideas?

Thank you for your time and I look forward to hearing from you.

**Comment by rkazak** [comment]
I would be interested in this!  I would look at what some diff/merge tools present as a gui to get ideas!
